packages/orchestrator-ui-components/src/components/WfoRadioDropdown/WfoRadioDropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+
+import {
+    EuiButtonEmpty,
+    EuiFlexGroup,
+    EuiPopover,
+    EuiRadioGroup,
+} from '@elastic/eui';
+import { EuiRadioGroupOption } from '@elastic/eui/src/components/form/radio/radio_group';
+
+import { WfoRadioDropdownOption } from '@/types';
+
+export interface WfoRadioDropdownProps<T> {
+    options: WfoRadioDropdownOption<T>[];
+    selectedOption: WfoRadioDropdownOption<T>;
+    onUpdateOption: (value: WfoRadioDropdownOption<T>) => void;
+}
+
+export const WfoRadioDropdown = <T,>({
```

Comment:
Is the , a typo?
```suggestion
export const WfoRadioDropdown = <T>({
```

Reply:
Not a typo. Otherwise the compiler thinks it is a functional component <T>.